marimbabot_speech/src/text2speech.py

```diff
@@ -35,28 +31,11 @@ def callback_response(self, response):
             sound_request.sound = -3
             sound_request.arg = text
 
-            # # play sound from audio file
-            # sound_request.sound = -2
-
-            # temp_ = tempfile.TemporaryFile()
-            # audio_filename = str(temp_.name) + ".wav"
-
-            # with open(audio_filename, "w+") as f:
-            #     mimic_subprocess = subprocess.Popen(
-            #         ('mimic3', '--voice', self.voice, '--length-scale', self.speed, text), 
-            #         # stdout=subprocess.PIPE)
-            #         stdout=f)
-
-            #     # Wait for the process to finish
-            #     mimic_subprocess.wait()
-
-            # sound_request.arg = audio_filename
-
         # publish audio file to audio node (sound_play package)
         self.audio_pub.publish(sound_request)
 
 if __name__ == '__main__':
-    rospy.init_node('speech_synthesis_node')
+    rospy.init_node('speech_stt_node')
```

Comment:
stt=speechtotext? maybe rename the node

Reply:
yeah, actually it is a standard abbrevation, it is commonly used. perhaps I can declarify it somewhere.